extract_unique_sensors_01.py

```diff
@@ -16,7 +19,12 @@ def create_parser():
     parser.add_argument('--url', help='A reference to SensorList.txt that specifies its ' 
                         'location on a computer network.') 
     parser.add_argument('--kat_sensor', required=True, help='Name of unique sensor')
+    parser.add_argument('-v', metavar='verbosity', type=int, default=1,
+    help='Verbosity of logging: 0 -critical, 1- error, 2 -warning, 3 -info, 4 -debug')
+    
     args = parser.parse_args()
+    verbose = {0: logging.CRITICAL, 1: logging.ERROR, 2: logging.WARNING, 3: logging.INFO, 4: logging.DEBUG}
+    logging.basicConfig(format='%(message)s', level=verbose[args.v], stream=sys.stdout)
```

Comment:
Why are you importing all logging levels and only using the `error` in your code?